lib/Cargo.toml

```diff
@@ -37,8 +37,8 @@ anyhow = "1.0"
 log = "0.4.20"
 once_cell = "1.19"
 thiserror = "1.0"
-# Version must match that used by uniffi-bindgen-go
-uniffi = "0.25.0"
+uniffi_25 = { package = "uniffi", version = "0.25.2" }
+uniffi_28 = { package = "uniffi", version = "0.28.0" }
```

Comment:
Do we need this at the workspace level? Or can be scoped to bindings crate only?

Reply:
No, I moved them